e2e/tests/ui-driven/src/helpers/context.ts

```diff
@@ -50,6 +50,24 @@ export async function setUpTestContext(
         submissionEmail: context.team.settings?.submissionEmail,
       },
     });
+    const integrations = await $admin.client.request(
```

Comment:
Couldn't find a better way to switch `has_planning_data` to `true` - I'll probably move the mutation out and add it to the `beforeAll` for the geospatial stuff since it's isolated to those tests